lib/components/modal.vue

```diff
@@ -69,20 +69,32 @@
         opacity: 0 !important;
     }
 
-    /* Make modal display as block instead of inline style, and because Vue's v-show deletes inline "display" style*/
+    /* Make modal display as block instead of inline style, and because Vue's v-show deletes inline "display" style */
     .modal {
         display: block;
     }
 </style>
 
 <script>
+    import generateId from '../mixins/generate-id';
     import bBtn from './button.vue';
 
+    const SELECTOR = [
+        'button:not([disabled])',
+        'input:not([disabled])',
+        'select:not([disabled])',
+        'textarea:not([disabled])',
+        'a:not([disabled]):not(.disabled)',
+        '[tabindex]:not([disabled]):not(.disabled)'
+    ].join(',');
+
     export default {
+        mixins: [generateId],
```

Comment:
Ah, yes.  I'll remove that, and make sure `aria-labeledby` and `aria-describedby` don't get set if the ID is missing.